### PR TITLE
5.4.x

### PR DESCRIPTION
**Pull request by fdbiondi**
PHP 5.4, MySql 5.6

### CODE REVIEW

**Comment by MrOffline77** [comment]
Hi thank you for pull request. I will take a look at this at the weekend.

File: config/php/php.ini [hunk 1]
```diff
@@ -1,3 +1,5 @@
-memory_limit = 128M
+memory_limit = 1G
 post_max_size = 100M
 upload_max_filesize = 20M
+date.timezone=America/New_York
```
Comment (narendravaghela) [inline config/php/php.ini]: It should be `UTC`.

**Comment by serbanghita** [comment]
Update and ship it

**Comment by MrOffline77** [comment]
* resolved merge conflict in php54 branch
* I'm running a test build , when it works, I will do the php.ini fixes by my self, otherwise we wont go anywhere here.

**Comment by MrOffline77** [comment]
test ok

**Pull request merged**